lib/platform/vsts/index.js

```diff
@@ -0,0 +1,413 @@
+// @ts-check
+const vstsHelper = require('./vsts-helper');
+const gitApi = require('./vsts-got-wrapper');
+
+const config = {};
+
+module.exports = {
+  // Initialization
+  getRepos,
+  initRepo,
+  setBaseBranch,
+  // Search
+  getFileList,
+  // Branch
+  branchExists,
+  getAllRenovateBranches, // Not implemented yet
+  isBranchStale, // Not implemented yet
+  getBranchPr,
+  getBranchStatus,
+  getBranchStatusCheck,
+  setBranchStatus,
+  deleteBranch, // Not implemented yet
```

Comment:
Mostly just important for cleanup, but should be a fairly simple implementation?